src/bsp/src/stm32/src/BSP.cpp

```diff
@@ -3,6 +3,22 @@
 #include "bsp/BSPContainer.h"
 #include <hal/hal.h>
 #include <logger/LoggerContainer.h>
+#include <Task.h>
+
+#ifdef RUNTIME_STATS
+#include "task.h"
+void usageFunctionCallback(void* ctx) {
+    (void)ctx;
+    char buffer[1000];
```

Comment:
should we make a big buffer like this static? Seems like a lot of memory to put on the stack. (Or actually the heap as the task is not created as static)

Reply:
Done hough we will still need to have a stack of at least 1000 since the logger creates a buffer of 1000 when printing